notebooks/templates/ai_platform_notebooks_template.ipynb

```diff
@@ -1,524 +1,634 @@
 {
```

Comment:
<p>Is there a spacing issue around following the guidelines?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/167/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>Existing cell, I made no changes -- can't see the spacing issue?</p>